cmd/kor/configmaps.go

```diff
@@ -14,7 +14,7 @@ var configmapCmd = &cobra.Command{
 	Args:    cobra.ExactArgs(0),
 	Run: func(cmd *cobra.Command, args []string) {
 		clientset := kor.GetKubeClient(kubeconfig)
-		if response, err := kor.GetUnusedConfigmaps(includeExcludeLists, clientset, outputFormat, slackOpts); err != nil {
+		if response, err := kor.GetUnusedConfigmaps(includeExcludeLists, clientset, outputFormat, slackOpts, deleteOpts); err != nil {
```

Comment:
nit: to avoid having to change all these functions every time we need more data to be passed, we merge the `slackOpts` and `deleteOpts` into an opts struct that would carry all the info, you can even compose it from the existing structs